x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/chunking/ChunkerBuilderTests.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference.chunking;
+
+import org.elasticsearch.inference.ChunkingStrategy;
+import org.elasticsearch.test.ESTestCase;
+
+import java.util.Map;
+
+public class ChunkerBuilderTests extends ESTestCase {
+
+    public void testNullChunkingStrategy() {
+        assert (ChunkerBuilder.fromChunkingStrategy(null) instanceof WordBoundaryChunker);
```

Comment:
We tend to use the Hamcrest matchers and assertThat rather than raw asserts

```suggestion
        assertThat(ChunkerBuilder.fromChunkingStrategy(null), instanceOf(WordBoundaryChunker));
```